openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -346,6 +347,18 @@ def sync_project_from_kitsu(dbcon: AvalonMongoDB, project: dict):
     if not project:
         project = gazu.project.get_project_by_name(project["name"])
 
+    # Get all statuses for projects from Kitsu
+    all_status = gazu.project.all_project_status()
+    for status in all_status:
+        if project['project_status_id'] == status['id']:
+            project['project_status_name'] = status['name']
+
+    if (
+        project['project_status_name'] == "Closed"
+        and not get_project(project['name'])
+    ):
```

Comment:
To make it more explicit I'd switch statements and add a comment.
```suggestion
    # Do not create new project if closed
    if (
        not get_project(project['name'])
        and project['project_status_name'] == "Closed"
    ):
```

Reply:
This would be slower however for all open projects since it'd first touch the OpenPype database. If kept as is then it only queries the OpenPype databse if the project is closed.

Reply:
That is from my suggestion https://github.com/ynput/OpenPype/pull/4619#discussion_r1154242344 to avoid query of project all the time.

Reply:
Ah you're right, disregard the switch then, but keep the comment, pick the one you prefer between @BigRoy's or mine ;)